src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,138 @@
+package duke;
+
+/**
+ * TO DO:
+ * Update EXPECTED.TXT
+ */
+
+import duke.command.Command;
+import duke.command.ExecutedCommand;
+import duke.tasks.*;
```

Comment:
Ideally, all imported classes must be listed out explicitly. No wildcard imports, please!